src/lightning_lite/strategies/launchers/multiprocessing.py

```diff
@@ -176,3 +179,22 @@ def restore(self) -> None:
 def _is_forking_disabled() -> bool:
     """Returns whether forking is disabled through the environment variable ``PL_DISABLE_FORK``."""
     return bool(int(os.environ.get("PL_DISABLE_FORK", "0")))
+
+
+def _check_bad_cuda_fork() -> None:
+    """Checks whether it is safe to fork and initialize CUDA in the new processes, and raises an exception if not.
+
+    The error message replaces PyTorch's 'Cannot re-initialize CUDA in forked subprocess' with helpful advice for
+    Lightning users.
+    """
+    if not torch.cuda.is_initialized():
+        return
+
+    message = (
+        "Lightning can't create new processes if CUDA is already initialized. Did you manually call"
+        " `torch.cuda.*` functions, have moved the model to the device or allocated memory on the GPU any"
```

Comment:
oxford comma :tophat: 

```suggestion
        " `torch.cuda.*` functions, have moved the model to the device, or allocated memory on the GPU any"
```